dashboard/config/locales/en.yml

```diff
@@ -172,6 +172,10 @@ en:
     student_consent: "I confirm that I have my parent or legal guardian's permission to use the Code.org services."
     agree_us_website: "I agree to using a website based in the United States."
     my_data_to_us: "Data from my use of this site may be sent to and stored or processed in the United States."
+    agree_us_data_transfer_student: '<strong>I agree that Code.org may transfer data (including personal data) from my use of this site to the United States</strong> for the purpose of hosting and processing such data.'
+    agree_us_data_transfer_teacher: '<strong>I agree that Code.org may transfer data (including personal data) from my use (and my students’ use) of this site to the United States</strong> for the purpose of hosting and processing such data.'
+    data_collection_us: "I understand that Code.org is a US-based not-for-profit website and that the laws governing data collection in the U.S. may differ from the laws in my country."
+    visit_privacy: 'Visit Code.org’s <a href="%{privacy_url}" target="_blank">Privacy Policy</a> to learn more.'
```

Comment:
Please use markdown rather than HTML for formatting when possible.

Also note that opening links in a new tab is only recommended if the page the link is being rendered on has some state that the user might lose if they navigate away (like a code editor), and even in that case we need to take extra security measures to attempt to reduce (but not eliminate) the risk of hostile takeover. What page is the `visit_privacy` string being rendered on?

Reply:
The visit_privacy string will be rendered on the "Finish creating your account" page (the 2nd page of the sign up flow). Looks like the links we have there to the tos and privacy policy open in the same tab, even though the user could lose the information they previously entered into the form. I'm happy to simply follow that precedent if you believe that's the best choice.

Reply:
I think I would prefer to follow the precedent; this is the case where I think we could reasonably go either way, so I'm gonna vote we default to the more-secure option.